sections/clause_06_overview.adoc

```diff
@@ -144,10 +146,7 @@ SensorThings API uses the term of Sensor to describe the Observer that is used i
 
 === Relation to OASIS-OData
 
-The OGC SensorThings API v2 interface is not an OData interface.
-It specifies a subset of the OData interface, and extends it at the same time.
-
-An SensorThings API Server implementation can implement the full OData specification.
-An OData client can access a SensorThings API service.
+The OGC SensorThings API v2 interface is not an OData interface and does not claim to be an OData service. It specifies a subset of the OData interface, and extends it at the same time.
```

Comment:
Git works on a per-line-basis, so avoid long lines. Once sentence per line gives the cleanest history and makes diffs easier to read.